core/accounting_test.go

```diff
@@ -0,0 +1,92 @@
+package core
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEmptyb_ReturnsZeroedValues(t *testing.T) {
```

Comment:
"Emptyb" -> "EmptyBalances"?

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/956/commits/794632acb80531d8222ca2a12334d3ddcd321427